collector-sqs/src/main/java/zipkin/collector/sqs/SqsCollector.java

```diff
@@ -0,0 +1,108 @@
+package zipkin.collector.sqs;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.amazonaws.services.sqs.AmazonSQSClient;
+import zipkin.collector.Collector;
+import zipkin.collector.CollectorComponent;
+import zipkin.collector.CollectorMetrics;
+import zipkin.collector.CollectorSampler;
+import zipkin.internal.Util;
+import zipkin.storage.StorageComponent;
+
+import java.io.Closeable;
+import java.io.IOException;
+
+public class SqsCollector implements CollectorComponent, Closeable {
+
+    private Collector collector;
+    private CollectorMetrics metrics;
+
+    private AWSCredentialsProvider credentialsProvider;
+    private String sqsQueueUrl;
+    private int waitTimeSeconds;
+
+    private AmazonSQSClient client;
+
+    public SqsCollector(Builder builder) {
+        collector = builder.delegate.build();
+        metrics = builder.metrics;
+        credentialsProvider = builder.credentialsProvider;
+        sqsQueueUrl = builder.sqsQueueUrl;
+        waitTimeSeconds = builder.waitTimeSeconds;
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static final class Builder implements CollectorComponent.Builder {
+        Collector.Builder delegate = Collector.builder(SqsCollector.class);
+        CollectorMetrics metrics = CollectorMetrics.NOOP_METRICS;
+
+        String sqsQueueUrl;
+        int waitTimeSeconds = 1;
```

Comment:
What are your thoughts on making this 20 seconds?  The SQS API can be configured to hold the request open for 20 seconds waiting for messages.  That way if there are no spans it will only make an API call ever 20 seconds instead of every second.  Some cases where this might be nice are for dev or staging environments with lower traffic.